pkg/gadgets/dns/tracer/tracer.go

```diff
@@ -167,7 +167,101 @@ var pktTypeNames = []string{
 	"KERNEL",
 }
 
-func parseDNSEvent(rawSample []byte) (ret string, pktType string) {
+// List taken from:
+// https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-4
+var qTypeNames = []string{
+	1:     "A",
```

Comment:
I am just wondering if we should use `A-IPv4` and `AAAA-IPv6` or maybe just `IPv4` and `IPv6`. Not everybody knows  `A` and `AAAA` means that.

Reply:
I think people are familiar enough with `A` and `AAAA`. I won't deviate from the standard representation of such queries.

Reply:
As a middle ground, the last version of the patch shows `A (IPv4)` and `AAAA (IPv6)`.

Reply:
I still think `A` and `AAAA` are better, it's the standard and I don't see any reason why to use something different. Additionally IMO `A` and `AAAA` are widely known and we should not worry about Jose's concern above.

Reply:
I finally went with the standard.
It is possible people will not know it but it is not so complicated to learn superficially to understand `A` and `AAAA`.